tests/functional/metadata/mongodb/listLifecycleObject/current.spec.js

```diff
@@ -0,0 +1,409 @@
+const async = require('async');
+const assert = require('assert');
+const werelogs = require('werelogs');
+const { MongoMemoryReplSet } = require('mongodb-memory-server');
+const logger = new werelogs.Logger('MongoClientInterface', 'debug', 'debug');
+const MetadataWrapper =
+require('../../../../../lib/storage/metadata/MetadataWrapper');
```

Comment:
Not sure why the linter isn't picking up, this line should be indented for readablity.